devtools/gotemplatelinter/final-newline-rule.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
We do not use hyphen in the filename of Go source files. Use underscore instead.

Reply:
Fixed up in 443f77e8186a454a369938ee5a8d6b3cdd0f6620